test/integration/common.c

```diff
@@ -41,14 +41,17 @@ create_primary (TSS2_SYS_CONTEXT *sapi_context,
                 TPM2_HANDLE       *handle)
 {
     TSS2_RC rc;
-    TPM2B_SENSITIVE_CREATE in_sensitive    = { 0 };
-    TPM2B_PUBLIC           in_public       = { 0 };
-    TPM2B_DATA             outside_info    = { 0 };
-    TPML_PCR_SELECTION     creation_pcr    = { 0 };
-    TPM2B_PUBLIC           out_public      = { 0 };
-    TPM2B_CREATION_DATA    creation_data   = { 0 };
+    TPM2B_SENSITIVE_CREATE in_sensitive    = { 0, { { 0, 0 }, { 0, 0 } } };
+    TPM2B_PUBLIC           in_public       = { 0, { 0, 0, 0, { 0, 0 }, { 0,
+        { 0 } }, { 0, 0 } } };
+    TPM2B_DATA             outside_info    = { 0, 0 };
+    TPML_PCR_SELECTION     creation_pcr    = { 0, { 0, 0, 0 } };
+    TPM2B_PUBLIC           out_public       = { 0, { 0, 0, 0, { 0, 0 }, { 0,
+        { 0 } }, { 0, 0 } } };
+    TPM2B_CREATION_DATA    creation_data   = { 0, { { 0, { 0, 0, 0 } },
+        { 0, 0 }, 0, 0, { 0, 0 }, { 0, 0 }, { 0, 0 } } };
     TPM2B_DIGEST           creation_digest = TPM2B_DIGEST_STATIC_INIT;
-    TPMT_TK_CREATION       creation_ticket = { 0 };
+    TPMT_TK_CREATION       creation_ticket = { 0, 0, { 0, 0 } };
```

Comment:
init macros please, pretty please 🙏

Reply:
Again, the init macros belong in a separate repository, but I can add them here in abrmd instead.